homebrew_challenge/src/components/Facts/Facts.css

```diff
@@ -0,0 +1,71 @@
+.icons-row {
```

Comment:
Seems a bit of a generic classname (ie risk it being used elsewhere) - either rename it soe something facts specific, or specify the containing class

Reply:
removed - I wasn't using anymore - realised when I went back to update it.